bundle/Resources/config/services.yaml

```diff
@@ -0,0 +1,74 @@
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\Service\ScheduledVisibilityService:
+        arguments:
+            - '@Netgen\Bundle\IbexaScheduledVisibilityBundle\Configuration\ScheduledVisibilityConfiguration'
+            - '@Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Registry'
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Registry:
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler\Content:
+        arguments:
+            - '@ibexa.api.repository'
+            - '@ibexa.api.service.content'
+        tags:
+            - { name: netgen.ibexa_scheduled_visibility.handler, identifier: content }
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler\Location:
+        arguments:
+            - '@ibexa.api.repository'
+            - '@ibexa.api.service.location'
+        tags:
+            - { name: netgen.ibexa_scheduled_visibility.handler, identifier: location }
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler\ContentAndLocation:
+        arguments:
+            - '@Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler\Content'
+            - '@Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler\Location'
+        tags:
+            - { name: netgen.ibexa_scheduled_visibility.handler, identifier: content_location }
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler\Section:
+        arguments:
+            - '@ibexa.api.repository'
+            - '@ibexa.api.service.section'
+            - '%netgen_ibexa_scheduled_visibility.sections.hidden.section_id%'
+            - '%netgen_ibexa_scheduled_visibility.sections.visible.section_id%'
+        tags:
+            - { name: netgen.ibexa_scheduled_visibility.handler, identifier: section }
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Handler\ObjectState:
+        arguments:
+            - '@ibexa.api.repository'
+            - '@ibexa.api.service.object_state'
+            - '%netgen_ibexa_scheduled_visibility.object_states.object_state_group_id%'
+            - '%netgen_ibexa_scheduled_visibility.object_states.hidden.object_state_id%'
+            - '%netgen_ibexa_scheduled_visibility.object_states.visible.object_state_id%'
+        tags:
+            - { name: netgen.ibexa_scheduled_visibility.handler, identifier: object_state }
+
+    Netgen\Bundle\IbexaScheduledVisibilityBundle\Command\UpdateContentVisibilityCommand:
+        arguments:
+            - '@ibexa.api.repository'
+            - '@Netgen\Bundle\IbexaScheduledVisibilityBundle\Service\ScheduledVisibilityService'
+            - '@Netgen\Bundle\IbexaScheduledVisibilityBundle\Configuration\ScheduledVisibilityConfiguration'
+            - '@ibexa.persistence.connection'
+            - '@?logger'
+        tags:
+            - { name: console.command, command: 'ngscheduledvisibility:content:update-visibility' }
```

Comment:
Rename command handle to `ngscheduledvisibility:update`.

Reply:
fixed in 477055737868a2001a2c1b07bf29d399a0fed97e